stl/inc/deque

```diff
@@ -1571,6 +1575,18 @@ template <class _Iter, class _Alloc = allocator<_Iter_value_t<_Iter>>,
 deque(_Iter, _Iter, _Alloc = _Alloc())->deque<_Iter_value_t<_Iter>, _Alloc>;
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+template <class _Ty, class _Alloc, class _Pr>
+void erase_if(deque<_Ty, _Alloc>& _Cont, _Pr _Pred) {
+    _Erase_remove_if(_Cont, _Pass_fn(_Pred));
+}
+
+template <class _Ty, class _Alloc, class _Uty>
+void erase(deque<_Ty, _Alloc>& _Cont, const _Uty& _Val) {
+    _Erase_remove(_Cont, _Val);
+}
+#endif

```

Comment:
We conventionally comment all `#else` and `#endif` directives (with very few exceptions). This makes it easier to see the preprocessor logic. We try to follow the convention even for short, simple regions, because code often expands later. Please check all occurrences you're adding.